<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CppTools::ClangDiagnosticConfigsWidget</class>
 <widget class="QWidget" name="CppTools::ClangDiagnosticConfigsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>665</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QComboBox" name="configChooserComboBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="copyButton">
       <property name="text">
        <string>Copy...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="removeButton">
       <property name="text">
        <string>Remove</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="infoIcon">
         <property name="text">
          <string>InfoIcon</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="infoLabel">
         <property name="text">
          <string>InfoText</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTabWidget" name="tabWidget"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>configChooserComboBox</tabstop>
  <tabstop>copyButton</tabstop>
  <tabstop>removeButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
